app/models/hcm.rb

```diff
@@ -10,7 +10,7 @@ class Hcm < ActiveRecord::Base
     'country' => { column: :country, converter: BaseConverter },
     'institution type' => { column: :institution_type, converter: BaseConverter },
     'stop pay/monitor method' => { column: :hcm_type, converter: BaseConverter },
-    'method reason desc' => { column: :hcm_reason, converter: DisplayConverter }
+    'method reason desc' => { column: :hcm_reason, converter: BaseConverter }
```

Comment:
Yes - Patrick's issue was with the case of letters in acroynyms being an issue with the Dept of Ed. The BaseConverter leaves the case intact, but does strip quote characters, `null`, `none` and `privacysuppressed`, as well leading and trailing blanks. We are now displaying caution flag reasons exactly as input.